Example/RSFormViewExampleApp/SignupExampleViewController.swift

```diff
@@ -30,14 +30,14 @@ class SignupExampleViewController: UIViewController {
     let configurator = FormConfigurator()
     let darkPurple = UIColor.formColor(red: 140, green: 20, blue: 252)
```

Comment:
Is this still being used? If not, this should be deleted

Reply:
good catch, missed this!